### PR TITLE
Remove SDK Modes content from Go-SDK Integration Page 

### LINKED ISSUE #54: [ Documentation request ] Remove SDK Modes content from Go-SDK Integration Page
# Brief description

When you visit [`Go SDK/ Integration`](https://docs.keploy.io/docs/go/installation) page you'll see the SDK modes content, which already has a dedicated page to it under [`Docs/Concept/Keploy SDK Modes`](https://docs.keploy.io/docs/concepts/what-are-keploy-sdk-modes). So, rather than importing the SDK Modes part in the Integration page we can just add a redirection link and delete the irrelevant content.

# Your recommended content

Refer to [this file](https://github.com/keploy/docs/blob/main/docs/go/installation.md) for changes.

### PR DESCRIPTION
**Pull request by Manoj0Marmat**
Signed-off-by: Manoj Marmat <manojrajput762000@gmail.com>

# Pull Request Template

## Description

Please include a summary of the change. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #54 
Screenshot 2022-10-08 at 10 27 56 AM

## Type of change

Please delete options that are not relevant.

- [x] This change requires a documentation update.

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Additional Context (Please include any Screenshots/gifs if relevant)

...

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have made corresponding changes to the documentation.
- [x] I have added corresponding tests.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by Manoj0Marmat** [comment]
@nehagup @SanskritiHarmukh issue #55 java sdk doc indentation problem forgot to create another branch please review it 
Screen Recording 2022-10-08 at 10 48 08 AM

File: docs/go/installation.md [hunk 1]
```diff
@@ -21,9 +21,9 @@ Or clone the Go SDK repo to your preferred location:
 git clone git@github.com:keploy/go-sdk.git
 ```
 
-import KeploySDKModes from '../concepts/what-are-keploy-sdk-modes.md'
+### SDK Modes
 
-<KeploySDKModes/>
+Refer **_ [this](../concepts/what-are-keploy-sdk-modes.md) _** article for SDK Modes.
```
Comment (SanskritiHarmukh) [inline docs/go/installation.md:26]: You can reform this line to - "Learn more about SDK modes" and attach link to SDK modes.

Comment (Manoj0Marmat) [reply docs/go/installation.md:26]: okay got it

File: docs/java/installation.md [hunk 2]
```diff
@@ -60,81 +60,82 @@ or to _build.gradle_:
...
+                             <dataFile>target/jacoco.exec</dataFile>
+                             <!-- Sets the output directory for the code coverage report. -->
+                             <outputDirectory>target/my-reports<outputDirectory>
+                         </configuration>
+                     </execution>
+                 </executions>
+             </plugin>
+         ```
+
+      5. Run your tests using command : `mvn test`.
```
Comment (SanskritiHarmukh) [inline docs/java/installation.md:138]: This LGTM.

**Comment by Manoj0Marmat** [comment]
@SanskritiHarmukh @nehagup right now just fixing go sdk formatting i'll make another pr of java sdk and other doc issue ..

**Pull request closed without merging**